pkg/webui/console/components/rights-group/rights-group.styl

```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-.id
-  h2()
+.select-all
+  margin-bottom: $ls.xs
 
```

Comment:
The hierarchical relationship between the `All Rights` checkbox and their subordainates needs to be made clearer visually. I think applying a left pading to the checkbox group will help here:
```
    border-normal('left')
    padding-left: $ls.m
```
![image](https://user-images.githubusercontent.com/5710611/59607550-d2e2dd00-9113-11e9-845b-315f7562447a.png)